CommunityToolkit.Mvvm/ComponentModel/ObservableValidator.cs

```diff
@@ -802,6 +802,7 @@ private void ClearErrorsForProperty(string propertyName)
     /// </summary>
     /// <param name="propertyName">The target property name being validated.</param>
     /// <returns>The display name for the property.</returns>
+    [RequiresUnreferencedCode("The type of the current instance cannot be statically discovered.")]
```

Comment:
```suggestion
    [RequiresUnreferencedCode("The current instance type cannot be statically discovered.")]
```

How about this?

Reply:
This is a standard error message also used in many APIs in the BCL, I want it to be consistent.